token/request.go

```diff
@@ -481,8 +502,23 @@ func (t *Request) Inputs() (*InputStream, error) {
 }
 
 func (t *Request) inputs(failOnMissing bool) (*InputStream, error) {
+	if t.TokenService == nil || t.TokenService.tms == nil {
```

Comment:
if these checks fail, we have a big problem no? Either something is not properly initialized, therefore a bug must be fixed, or the system is not responding as expected. Does it make sense to have a function to validate the metadata and that can be used once before any other operation is performed?

Reply:
I agree - from the code, this will never happen.